main.go

```diff
@@ -51,7 +51,20 @@ func main() {
 		fmt.Println(err)
 		os.Exit(1)
 	}
-	summary := reporter.Stream(config, results)
+
+	err = reporter.Start(config)
+	if err != nil {
```

Comment:
nit: use `err := reporter.Start` and inline into if. that way the `err` is scoped only to the body of the `if`